Mytinerary/Controller/ProfileViewController.m

```diff
@@ -30,6 +31,8 @@ - (void)viewDidLoad {
     self.collectionView.dataSource=self;
     self.collectionView.delegate=self;
     
+    // insert profile cell
```

Comment:
great way to track these kinds of TODOs is to create an issue on Github and link to that issue number in this comment

Reply:
Do you mean link as in referencing the issue number in the comment in the code itself, or is there a way to actually link the two with github?